src/CLI/Progress/ProgressBarProgress.php

```diff
@@ -15,11 +15,9 @@
  */
 class ProgressBarProgress implements Progress
 {
-    /** @var OutputInterface */
-    private $output;
+    private \Symfony\Component\Console\Output\NullOutput|OutputInterface $output;
```

Comment:
isn't NullOutput implementing OutputInterface? If yes, that could be just 

`private OutputInterface $output;`